### ISSUE TITLE
diff does not work on mac

### ISSUE THREAD

**Issue by builder555** [zpm-zsh/colorize#12]
Steps to reproduce:

- Install colorize on MacOS
- run `diff . /`

Expected results:  
  a list of files

Actual results:  
An error message:
```
diff: unrecognized option `--color'
diff: Try `diff --help' for more information.
```

**Comment by Freed-Wu** [comment]
Maybe macOS use bsd diff not GNU diff? Does there have a program named `gdiff` is GNU diff? Can you provide a PR about it?

**Comment by Freed-Wu** [comment]
Fixed by #11
